arch/arm/src/cxd56xx/cxd56_nxaudio.c

```diff
@@ -2698,7 +2699,7 @@ static int cxd56_getcaps(struct audio_lowerhalf_s *lower, int type,
  *
  ****************************************************************************/
 
-static int cxd56_shutdown(struct audio_lowerhalf_s *lower)
+static int cxd56_shutdown(struct audio_lowerhalf_s *lower, int opencnt)
```

Comment:
In the commit message you said "refcount" but here you used "opencnt" and in the classd_shutdown() you used only "cnt". I think it is a good idea to use just a single name here.